.envrc

```diff
@@ -0,0 +1,2 @@
+export DJANGO_SECRET_KEY=beer
+export DATABASE_URL=postgres://postgres:beer@172.17.0.2:5432/postgres
```

Comment:
I assume you didn't mean to commit this...

Reply:
Now you know my password...

Reply:
shh... it's a secret